core/src/main/java/org/bitcoinj/utils/MonetaryFormat.java

```diff
@@ -60,12 +62,16 @@
     public static final String CODE_MBTC = "mBTC";
     /** Currency code for base 1/1000000 Bitcoin. */
     public static final String CODE_UBTC = "µBTC";
+    /** Currency symbol for base 1/100000000 Bitcoin. */
+    public static final String CODE_SAT = "sat";
     /** Currency symbol for base 1 Bitcoin. */
     public static final String SYMBOL_BTC = "\u20bf";
     /** Currency symbol for base 1/1000 Bitcoin. */
     public static final String SYMBOL_MBTC = "m" + SYMBOL_BTC;
     /** Currency symbol for base 1/1000000 Bitcoin. */
     public static final String SYMBOL_UBTC = "µ" + SYMBOL_BTC;
+    /** Currency symbol for base 1/100000000 Bitcoin (satoshi). */
```

Comment:
I don't think "sat" is a symbol. We probably don't have a symbol for the satoshi unit already. I suggest to remove this constant – you already added "sat" as a currency code.

Reply:
Thanks. Removed.